src/sentry/api/endpoints/group_hashes.py

```diff
@@ -42,3 +45,26 @@ def get(self, request, group):
             order_by='id',
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def delete(self, request, group):
+        id_list = request.GET.getlist('id')
+        if id_list is None:
+            return Response()
+
+        hash_list = GroupHash.objects.filter(
```

Comment:
yeah kind of like how we do deletions so we can hide it in the UI immediately (or change its visual state)